### DISCUSSION TITLE
Multiple Servers

### DISCUSSION THREAD

**Discussion by fuzunspm** [muety/wakapi#312] [Q&A]
Is it possible to define and use multiple wakapi api_url?

**Comment by muety** [comment]
Unfortunately, that is not something the CLI supports. However, there are multiple options that come close.

If you want to use Wakapi and WakaTime in parallel, you can use Wakapi's WakaTime integration that relays all incoming heartbeats on to WakaTime, so you have the data in both tools.

If you want to use two Wakapi instances in parallel, you would either have to wait for #105 to be implemented or, what you could also do is try set up a fordward proxy-like application that replicates requests to multiple downstreams. I haven't done that, yet, though.
